programs/ssl/ssl_server2.c

```diff
@@ -1430,16 +1430,16 @@ int dummy_ticket_parse(void *p_ticket, mbedtls_ssl_session *session,
         case 2:
             return MBEDTLS_ERR_SSL_SESSION_TICKET_EXPIRED;
         case 3:
-            session->start = mbedtls_time(NULL) + 10;
+            session->start = mbedtls_ms_time() + 10 * 1000;
             break;
         case 4:
-            session->start = mbedtls_time(NULL) - 10 - 7 * 24 * 3600;
+            session->start = mbedtls_ms_time() - 10 * 1000 - 7 * 24 * 3600 * 1000;
             break;
         case 5:
-            session->start = mbedtls_time(NULL) - 10;
+            session->start = mbedtls_ms_time() - 10 * 1000;
             break;
         case 6:
-            session->start = mbedtls_time(NULL);
+            session->start = mbedtls_ms_time();
```

Comment:
Though it's not the change of this PR, does it worth one description where is the **"%11"** from and what's the meaning of `1, 2, 3... `